src/libcalamaresui/utils/Paste.cpp

```diff
@@ -9,20 +9,35 @@
 
 #include "Paste.h"
 
+#include "Branding.h"
 #include "utils/Logger.h"
 
 #include <QFile>
 #include <QRegularExpression>
 #include <QTcpSocket>
 #include <QUrl>
+#include <QClipboard>
+#include <QApplication>
+#include <QStringList>
 
 namespace CalamaresUtils
 {
 
+QStringList UploadServersList = {
+    "fiche"
+    // In future more serverTypes can be added as Calamares support them
+    // "none" serverType is explicitly not mentioned here
+};
+
 QString
-sendLogToPastebin( QObject* parent, const QString& ficheHost, quint16 fichePort )
+sendLogToPastebin( QObject* parent )
 {
-    QString pasteUrlFmt = parent->tr( "Install log posted to:\n%1" );
+
+    const QString& ficheHost = Calamares::Branding::instance()->uploadServerURL();
+    quint16 fichePort = Calamares::Branding::instance()->uploadServerPort();
+
+    QString pasteUrlFmt = parent->tr( "Install log posted to\n\n%1\n\nLink copied to clipboard" );
```

Comment:
You can use rich-text formatting here, so it might be good to have something like <br/> and maybe even a clickable link. You would need to check if the layout / formatting comes out right, though.